app/policies/provider_policy.rb

```diff
@@ -27,6 +27,10 @@ def index?
     user.present?
   end
 
+  def new?
+    user.present?
+  end
+
```

Comment:
use alias_method and add tests please

Reply:
Done here: https://github.com/DFE-Digital/publish-teacher-training/pull/3159/commits/da9627faef3dbeb9c18f058cb20550cb6228fbcf

and here: https://github.com/DFE-Digital/publish-teacher-training/pull/3159/commits/5d5021ee9ae08d77da8ea8bae42231e4750cd20e